themes/default/content/docs/pulumi-cloud/cloud-rest-api/_index.md

```diff
@@ -2735,15 +2735,16 @@ Search for resources belonging to the given organization.
 
 | Name       | In    | Type          | Required | Description                                                                                                                              |
 |------------|-------|---------------|----------|------------------------------------------------------------------------------------------------------------------------------------------|
-| org        | path  | string        | true     | Name of the organization to search.                                                                                                      |
-| query      | query | string        | false    | The search query to execute. If omitted all resources are returned (subject to any pagination limits).                                   |
-| sort       | query | array[string] | false    | Results are returned sorted by this field value.                                                                                         |
-| asc        | query | boolean       | false    | Whether to return results in ascending or descending sort order.                                                                         |
-| size       | query | integer       | false    | How many results to return at a time.                                                                                                    |
-| page       | query | number        | false    | The page of results to return.                                                                                                           |
-| continue   | query | string        | false    | A continuation token for pagination.                                                                                                     |
-| facet      | query | array[string] | false    | If provided, an aggregation will be returned with the top-5 values for the given facet, along with how many resources have those values. |
-| properties | query | boolean       | false    | Whether to include resource properties in results. Not supported for all subscriptions.                                                  |
+| org        | path  | string           | true     | Name of the organization to search.                                                                                                      |
+| query      | query | string           | false    | The search query to execute. If omitted all resources are returned (subject to any pagination limits).                                   |
+| sort       | query | array[string]    | false    | Results are returned sorted by this field value.                                                                                         |
+| asc        | query | boolean          | false    | Whether to return results in ascending or descending sort order.                                                                         |
```

Comment:
If this param is optional, does that mean results are sorted _descendingly_ by default? (For example, if `sort` field is provided but this param isn't?)

Reply:
Yes, descending by default. Added clarification.